pkg/controllers/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -67,6 +68,7 @@ func NewConfigObserver(
 		oauth.ObserveIdentityProviders,
 		oauth.ObserveTemplates,
 		oauth.ObserveTokenConfig,
+		configobserveroauth.ObserveAccessTokenInactivityTimeout,
```

Comment:
you need to remove the field observation from the other observer, otherwise you've got two entities setting the same field

Reply:
Sure, will do. I am in the process of doing that.